snippets/standard/base-types/string-practices/cs/tostring.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Globalization;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        // <Snippet2>
+        string value = "The amount is " + 126.03.ToString(CultureInfo.InvariantCulture) + ".";
+        Console.WriteLine(value);
+        // </Snippet2>
+
+        // <Snippet1>
+        value = "The amount is " + 126.03 + ".";
```

Comment:
In the article text, this snippet goes first. As a result, in the article the first snippet doesn't declare the `value` variable, but the second declares. Might be confusing.

I would use two different `string` variables for different snippets: 
```csharp
string messageInInvariantCulture = ...
// and
string messageInCurrentCulture =...
```

Reply:
Thanks for noticing that, @pkulikov. I've reversed the order and declared separate string variables named `concat1` and `concat1`.